core/domain/user_jobs_one_off.py

```diff
@@ -626,3 +628,70 @@ def map(item):
     def reduce(key, values):
         values.sort()
         yield (key, values)
+
+
+class ProfilePictureAuditOneOffJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
Instead of having this be its own one-off job, do you think it would be better to have it be part of the prod validation job for UserSettingsModel instead? We are running periodic clean-ups of the one-off jobs in the codebase, so having it be part of the regular prod validation job might be better in order to ensure its longevity (unless it's intended to be a transient thing).

Reply:
@seanlip Some parts of this are transient I will use the data from this job to properly migrate the profile pictures to GCS. I would keep it separately for now and add some of this stuff to the validation job when migrating the profile pictures.